lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -1,5 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Performance: metrics evaluates valid input (with lcp) correctly 1`] = `
```

Comment:
let's make this observed so we test the `Ts` values coming through?

Reply:
huh? the test snapshots the entire metrics object.

Reply:
I keep getting `NO_TTI_CPU_IDLE_PERIOD` when I change `throttlingMethod` to `provided`. I tried updating the pwa.rocks m79 trace, and using cnn.com instead - still happens.

Reply:
This is blocking me.

Reply:
sorry! are you collecting with `--throttling-method=devtools`? we won't use a long enough quiet window with simulate for usage with observed metric methods

Reply:
I was doing `provided`. `devtools` worked.

Reply:
my observed changes were not necessary for httparchive, sorry @connorjclark :)